go-controller/pkg/metrics/ovn_db.go

```diff
@@ -390,7 +392,18 @@ func RegisterOvnDBMetrics(clientset kubernetes.Interface, k8sNodeName string) {
 	_, _, err = util.RunOVSDBTool("db-is-standalone", "/etc/openvswitch/ovnsb_db.db")
 	if err == nil {
 		dbIsClustered = false
+	} else {
```

Comment:
From our discussion on slack, I thought the root cause of this error was that the metrics code was attempting to access the /etc/openvswitch/ovnsb_db.db file before the database initialization was finished. Would it be better to have the metrics wait until the DB init has finished? 

I'm left wondering what effect this would have on a clustered system, if this error occurs and the `dbIsClustered` defaults to false. Do you see this on a clustered system?

Reply:
Turns out the mount point was incorrect. But as per @girishmg and I agree with him, this code shouldn't have been on the master in the first place. #1882 needs to get reverted.